src/com/zombiecastlerush/building/Castle.java

```diff
@@ -0,0 +1,42 @@
+package com.zombiecastlerush.building;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class Castle {
+    //create a list of rooms in castle
+    List<Room>  castleRooms = new ArrayList<Room>();
+
```

Comment:
how does the list provide an access point to this Castle?

Reply:
Aashis: Castle has a Map